src/main/resources/META-INF/plugin.xml

```diff
@@ -309,9 +309,9 @@
       <!-- component level -->
       <separator/>
       <group id="org.jboss.tools.intellij.openshift.actions.component.actions">
-        <action class="org.jboss.tools.intellij.openshift.actions.component.DevComponentAction" id="org.jboss.tools.intellij.openshift.actions.component.DevComponentAction"/>
-        <action class="org.jboss.tools.intellij.openshift.actions.component.DevOnPodmanComponentAction" id="org.jboss.tools.intellij.openshift.actions.component.DevOnPodmanComponentAction"/>
-        <action class="org.jboss.tools.intellij.openshift.actions.component.DeployComponentAction" id="org.jboss.tools.intellij.openshift.actions.component.DeployComponentAction"/>
+        <action class="org.jboss.tools.intellij.openshift.actions.component.DevComponentAction" id="org.jboss.tools.intellij.openshift.actions.component.DevComponentAction" text="Dev Component"/>
+        <action class="org.jboss.tools.intellij.openshift.actions.component.DevOnPodmanComponentAction" id="org.jboss.tools.intellij.openshift.actions.component.DevOnPodmanComponentAction" text="Dev Podman Component" />
+        <action class="org.jboss.tools.intellij.openshift.actions.component.DeployComponentAction" id="org.jboss.tools.intellij.openshift.actions.component.DeployComponentAction" text="Deploy Component" />
       </group>
```

Comment:
nitpicking but those labels are already defined on the ComponentFeature class with :
"Dev on Cluster"
"Dev on Podman"
"Deploy on Cluster"

Reply:
@sbouchet yes, but IDEA seems to impose those as the error message implies:
> The default action text must be specified in plugin.xml or its class constructor [Plugin: org.jboss.tools.intellij.openshift]
I dont see how we can reuse these with the plugin.xml or vice versa.

Reply:
yes but my point was more to reuse the same texts for default labels

Reply:
finally grocked :) corrected the labels.